diff --git a/simple_background_image_formatter.info.yml b/simple_background_image_formatter.info.yml
index 700511f..0f543ab 100644
--- a/simple_background_image_formatter.info.yml
+++ b/simple_background_image_formatter.info.yml
@@ -1,7 +1,6 @@
 name: Simple Background-image Formatter
 description: Provides a field formatter which allows images to be formatted as a css background-image instead of an actual html img tag.
 type: module
-core: 8.x
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^9.3 || ^10
 dependencies:
   - drupal:image
diff --git a/simple_background_image_formatter.module b/simple_background_image_formatter.module
index 6ef5a42..a052c90 100644
--- a/simple_background_image_formatter.module
+++ b/simple_background_image_formatter.module
@@ -48,7 +48,7 @@ function template_preprocess_simple_background_image_formatter(array &$variables
     $imageUrl = $style->buildUrl($uri);
   }
   else {
-    $imageUrl = file_create_url($uri);
+    $imageUrl = \Drupal::service('file_url_generator')->generateAbsoluteString($uri);
   }
 
   $variables['attributes']['style'] = "background-image: url(\"{$imageUrl}\");";
